gear/gear/profiling.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+
+import googlecloudprofiler
+
+HAIL_SHA = os.environ['HAIL_SHA']
+HAIL_SHOULD_PROFILE = os.environ.get('HAIL_SHOULD_PROFILE', False)
```

Comment:
You need to cast this to a boolean.

Reply:
Changed to an `in` check, since we only ever include that environment variable when we want to profile.